mappings/net/minecraft/world/updater/WorldUpdater.mapping

```diff
@@ -1,13 +1,20 @@
 CLASS net/minecraft/unmapped/C_jzcarjio net/minecraft/world/updater/WorldUpdater
```

Comment:
not a comment on this PR, but this should maybe be renamed to `WorldUpgrader` given that all the terminology inside is `upgrade` instead of `update`